apis/virtualkubelet/v1alpha1/shadowpod_types.go

```diff
@@ -24,15 +24,43 @@ type ShadowPodSpec struct {
 	Pod corev1.PodSpec `json:"pod,omitempty"`
 }
 
+// PodPhase describes the phase of the ShadowPod.
+type PodPhase string
+
+const (
+	// ShadowPodPending indicates a pending phase.
+	ShadowPodPending PodPhase = "Pending"
+	// ShadowPodRunning indicates running phase.
+	ShadowPodRunning PodPhase = "Running"
+	// ShadowPodSucceeded indicates succeeded phase.
+	ShadowPodSucceeded PodPhase = "Succeeded"
+	// ShadowPodFailed indicates failed phase.
+	ShadowPodFailed PodPhase = "Failed"
+	// ShadowPodUnknown indicates unknown phase.
+	ShadowPodUnknown PodPhase = "Unknown"
+)
```

Comment:
Can we re-use the corev1.PodPhase instead of re-define the type with its values?

Reply:
Done